drafts/Stormless3.md

```diff
@@ -0,0 +1,322 @@
+
+
+No?! Most likely, the [apig-s3 bug](https://github.com/sdd/serverless-apig-s3/issues/16), and you got to do a little final step manually. Open AWS Console, ->API Gateway->Resources->Actions-Deploy API. Or use AWS CLI:
+
+```
+# Find out your rest-api-id first
+aws apigateway get-rest-apis
+
+# Deploy the changes to the stage
+aws apigateway create-deployment --rest-api-id  1nort0aotd  --stage-name dev
+```
+
+Hope it works now.
+
+Congratulations! You made it. You might followed along and built your own app. You might drop it at some point of irritation. You might skimmed the text and examples. But if you made it to the end, you learned enough of serverless to be dangerous.
+
+
+As you gathered by now, serverless is not simple. Many things can go wrong as you build the app. You can blame some on me: I surely made a few mistakes and omissions (BTW reports and comments much appreciated!) But serverless complexity is not all my fault. We wire together many different services - AWS and 3rd party - with many different wires - frameworks and tools. The services are of different quality and [in]convenience. The framework and tools are still maturing.
+
+Why getting into serverless? Because for some classes of apps, it is substantially cheaper. This community signup is a case to point: we used to run StackStorm's with StackStorm. It was simple to build and solid like a rock, but took (before you get to multi-zone reliability). It didn't justify, given the expected amount of community signups. On the other extreme, high volume apps with high reliability requirements might be better off serverless for "infinite" elasticity which may be expensive to . I use "might" and "would" as it is not a clear cut and application specific. Do your math upfront, and keep watching the bill.
+
+As for taming complexity, Serverless framework helps. To fully appreciate it, try and redo the same app without it, as reliable, repeatable, reviewable infrastructure-as-code. While Serverless framework is not the only game in town, I specifically like it for pluggable architecture and ecosystem of plugins.
+
+Plugins help, picking up the areas not covered by Serverless core. We have enjoyed the simplicity of building StepFuncions and adding simple web front-ends (and if you haven't, try and redo it without the plugins). Skim and bookmark the [list of official plugins](https://github.com/serverless/plugins) to keep them in mind when for your next project.
+
+StackStorm Exchange - the recent addition to the serverless eco-system to - brings up a catalog of reusable integrations. It is not difficult to scrape APIs for ActiveCampaing or find and use undocumented Slack API calls. But it adds up to low-value routines, taking away from the focus on building the application. 
+
+<some good closing words> 
+
+
+### Final Tips and Tricks
```

Comment:
These last few tips are useful, but out of place. I would move these to the end of the previous section, and everything after the word "Congratulations!", I'd move down here to it's own "Conclusion" section. It's a great call-to-arms, and should be saved for the end.